### ISSUE TITLE
install guide references xinetd, but not installed or dependency

### ISSUE THREAD

**Issue by suchytas** [openhpc/ohpc#212]
Referring to Install Guide v1.0.1, Rev: 7e93115, Section 3.6 Complete basic Warewulf setup for master node

```
 # Restart/enable relevant services to support provisioning
 [sms]# systemctl restart xinetd
```

I installed with CentOS "Minimal" installation, and the xinetd services is not installed by default. To resolve, I had to install xinetd

`yum install xinetd`

Thinking about proposals.. 
- Make xinetd a dependency when installing ohpc-warewulf.. since this was part of the warewulf configuration
- Update the documentation with `yum install xinetd`
- or??

**Comment by koomie** [comment]
I think in this case, the dependency should be called out as part of Werewulf proper.  @crbaird, if you agree can you add to the relevant WW package?

**Comment by JohnWestlund** [comment]
Doesn't 'systemd' provide the necessary functionality, we're just not documenting enabling it via 'systemd'. Previously installing 'tftpd' would have pulled in 'xinetd', but now the assumption is you'd rely on the 'systemd'

**Comment by koomie** [comment]
I could be wrong, but I think Scott's original point was that he needed xinitd installed to get the relevant systemd file:

```
# rpm -ql xinetd | grep systemd
/usr/lib/systemd/system/xinetd.service
```

**Comment by JohnWestlund** [comment]
Yes -- but the only place we make use of `xinetd` in the install guide is to enable `tftp`. We could drop `xinetd` and just rely on `systemd`. I'm on board with using `xinetd` but that means we're resisting the new world order of `systemd`

**Comment by crbaird** [comment]
Thanks for catching this, @suchytas. It was patched for the CentOS 1.1 release in b0d8510 and just now for SLE in 1cca2c7. We will shortly issue an update for the 1.0.1 release.
